src/azul/chalice.py

```diff
@@ -619,8 +619,8 @@ def metric_alarms(self) -> Iterator[metric_alarm]:
             # first.
             for_errors = metric is LambdaMetric.errors
             alarm = self.metric_alarm(metric=metric,
-                                      threshold=1 if for_errors else 0,
-                                      period=24 * 60 * 60 if for_errors else 5 * 60)
+                                      threshold=0,
+                                      period=60 * 60 if for_errors else 5 * 60)
```

Comment:
Does this cause `datapoints_to_alarm` to be emitted?

Reply:
Please apply https://github.com/DataBiosphere/azul/pull/7358#discussion_r2291569610 to those places where `datapoints_to_alarm` is emitted.